controllers/datadogagent/component/utils.go

```diff
@@ -27,6 +30,15 @@ import (
 
 const (
 	localServiceDefaultMinimumVersion = "1.22-0"
+
+	DefaultAgentInstallType = "k8s_manual"
```

Comment:
how is this used? shouldn't this be `datadog-operator`?

Reply:
It's used for communicating to the Trace Agent the injection type that the client libraries were injected with. k8s_manual is the default value and will be overwritten by Cluster Agent if either local library injection or Single Step Instrumentation occur.

We only need this env variable until Agent 7.51.0 version is released, since we already merged a change to Trace Agent to handle properly the case when install type is not set.